tests/scripts/perf-prep.sh

```diff
@@ -60,25 +60,20 @@ curl "http://benchviewtestfeed.azurewebsites.net/nuget/FindPackagesById()?id='Mi
 unzip -q -o benchview.zip -d ./tests/scripts/Microsoft.BenchView.JSONFormat
 
 # Install python 3.5.2 to run machinedata.py for machine data collection
-python3.5 --version
-python3.5 ./tests/scripts/Microsoft.BenchView.JSONFormat/tools/machinedata.py
+python3 --version
+python3 ./tests/scripts/Microsoft.BenchView.JSONFormat/tools/machinedata.py
 
 if [ $throughput -eq 1 ]; then
     # Download throughput benchmarks
-    if [ -d "Microsoft.Benchview.ThroughputBenchmarks.x64.Windows_NT" ]; then
-        rm -r -f Microsoft.Benchview.ThroughputBenchmarks.x64.Windows_NT
-    fi
-    mkdir Microsoft.Benchview.ThroughputBenchmarks.x64.Windows_NT
-    cd Microsoft.Benchview.ThroughputBenchmarks.x64.Windows_NT
-
-    curl -OL https://dotnet.myget.org/F/dotnet-core/api/v2/package/Microsoft.Benchview.ThroughputBenchmarks.x64.Windows_NT/1.0.0
-    mv 1.0.0 1.0.0.zip
-    unzip -q 1.0.0.zip
+    if [ ! -d "Microsoft.Benchview.ThroughputBenchmarks.x64.Windows_NT" ]; then
```

Comment:
>if [ ! -d "Microsoft.Benchview.ThroughputBenchmarks.x64.Windows_NT" ]; then [](start = 4, length = 75)

Don't we want to delete the folder if it already exist? For example, it's a left over from a previous run. Then, create the folder?

Reply:
No, not with the way we are doing pipeline builds. In pipeline builds, we pull down the TP benchmarks on a windows machine and stash them. Then we will bring them over using unstash on the linux machine. So we don't want to delete the folder that we have just pulled down.